### PR TITLE
Extend Abracadabra Subgraph to use v2.0.1

### PR DESCRIPTION
**Pull request by nemani**
This PR attempts to extend Abracadabra Subgraph to support schema version > 2.0.0
Currently: 2.0.1

### CODE REVIEW

File: subgraphs/abracadabra/src/common/utils/arrays.ts [hunk 1]
```diff
@@ -1,3 +1,33 @@
+export function removeFromArrayAtIndex<T>(x: T[], index: i32): T[] {
```
Comment (this-username-is-taken) [inline subgraphs/abracadabra/src/common/utils/arrays.ts:1]: This is nice! Would you be able to add these to the `_reference_` subgraph?

File: subgraphs/abracadabra/src/mapping.ts [hunk 2]
```diff
@@ -193,53 +206,64 @@ export function handleLiquidation(event: LogRepay): void {
...
   liquidateEvent.amount = collateralAmount;
   liquidateEvent.amountUSD = collateralAmountUSD;
   liquidateEvent.profitUSD = collateralAmountUSD.minus(mimAmountUSD);
-  liquidateEvent.liquidatee = event.params.to.toHexString();
+  liquidateEvent.save();
+
+  let liqudidatedAccount = getOrCreateAccount(liquidateEvent.liquidatee!, event);
+  liqudidatedAccount.liquidateCount = liqudidatedAccount.liquidateCount + 1;
+  liqudidatedAccount.save();
+  addAccountToProtocol("LIQUIDATEE", liqudidatedAccount, event);
```
Comment (this-username-is-taken) [inline subgraphs/abracadabra/src/mapping.ts:225]: Probably better to make these named constants

**Comment by this-username-is-taken** [comment]
Everything looks good other than above. After you are done with the changes, would you be able to rename this `abracadabra-ext` while maintaining the 1.3.0 version as `abracadabra`? We want to maintain both versions going forward

File: subgraphs/abracadabra/src/common/utils/arrays.ts (hunk 1 above)
Comment (nemani) [reply subgraphs/abracadabra/src/common/utils/arrays.ts:1]: Sure, I will create another PR for this!

File: subgraphs/abracadabra/src/mapping.ts (hunk 2 above)
Comment (nemani) [reply subgraphs/abracadabra/src/mapping.ts:225]: done, in a new commit.

File: subgraphs/abracadabra/src/common/utils/arrays.ts (hunk 1 above)
Comment (nemani) [reply subgraphs/abracadabra/src/common/utils/arrays.ts]: https://github.com/messari/subgraphs/pull/493

**Pull request merged**